apps/egghead/src/app/api/og/[slug]/route.tsx

```
@@ -0,0 +1,355 @@
+export async function GET(
+	request: Request,
+	{ params }: { params: Promise<{ slug: string }> },
+) {
+	const corsHeaders = getCorsHeaders(request.headers.get('origin'))
+
+	try {
+		const { slug } = await params
+		console.log('Starting OG image generation for slug:', slug)
+
+		const { searchParams } = new URL(request.url)
+		const resourceTypesWithImages = [
```

Biome (2.1.2)

[error] Shouldn't redeclare 'OPTIONS'. Consider to delete it or rename it.

'OPTIONS' is defined here:


(lint/suspicious/noRedeclare)